docs/KUBERNETES.md

```diff
@@ -187,6 +187,27 @@ Alternatively, update the DaemonSet configuration directly by using the followin
 kubectl patch daemonset -n logdna-agent logdna-agent --type json -p '[{"op":"add","path":"/spec/template/spec/containers/0/securityContext/runAsUser","value":5000},{"op":"add","path":"/spec/template/spec/containers/0/securityContext/runAsGroup","value":5000}]'
 ```
 
+### Enabling persistent agent state
+
```

Comment:
My only issue - will users get "enabling persistent agent state" = "not omitting/duplicating logs?"? Otherwise I'd think fine. Thought this was a issue whether running as root or non-root.